src/riak_cs_storage.erl

```diff
@@ -100,8 +100,12 @@ sum_bucket(BucketName) ->
         ok = riak_cs_riak_client:set_bucket_name(RcPid, BucketName),
         {ok, ManifestPbc} = riak_cs_riak_client:manifest_pbc(RcPid),
         ManifestBucket = riak_cs_utils:to_bucket_name(objects, BucketName),
+        Input = case riak_cs_config:use_2i_for_storage_calc() of
+                    true -> {index, ManifestBucket, <<"$bucket">>, <<0>>, <<255>>};
```

Comment:
Probably caremad, but `<<255>>` is not the largest binary. For example, `riak_cs_list_objects_fsm_v2` uses 128 bytes of 255 ``binary:copy(255, 128). The one is also not the largest binary, but much safer.